mmeds/config.py

```diff
@@ -37,7 +37,10 @@
 MMEDS_LOG = DATABASE_DIR / 'mmeds_log.txt'
 SQL_LOG = DATABASE_DIR / 'sql_log.txt'
 DOCUMENT_LOG = DATABASE_DIR / 'document_log.txt'
-PROCESS_LOG = DATABASE_DIR / 'process_log.txt'
+PROCESS_LOG_DIR = DATABASE_DIR / 'process_log_dir'
+if not os.path.exists(PROCESS_LOG_DIR):
```

Comment:
not pathlib?!

Reply:
That is shameful. I just copied how I set up other directories back when I made this file at the beginning of mmeds